### PR TITLE
Enhancement/skale 3526 validation cmd

### PR DESCRIPTION
**Pull request by badrogger**

### CODE REVIEW

File: cli/validate.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+from core.host import validate_abi_files
+
+
+@click.group()
+def validate_cli():
+    pass
+
+
+@validate_cli.group(help="Resources allocation commands")
```
Comment (dmytrotkk) [inline cli/validate.py]: Fix help msg

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: cli/validate.py (hunk 1 above)
Comment (badrogger) [reply cli/validate.py]: addressed

**Review by dmytrotkk — APPROVED** [review]

File: core/host.py [hunk 2]
```diff
@@ -112,3 +115,18 @@ def safe_mk_dirs(path, print_res=False):
...
+        for abi_filepath in [
+            MANAGER_CONTRACTS_FILEPATH,
+            IMA_CONTRACTS_FILEPATH
+        ]
+    ]
+    if any(r['status'] == 'error' for r in results):
+        print('Some files are not exist or incorrect')
+        print_abi_validation_errors(results)
+    else:
+        print('Ok')
```
Comment (dmytrotkk) [inline core/host.py]: Maybe we should add a more informative message?
Something like `ABI files validated successfully!`

**Review by dmytrotkk — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -480,6 +483,23 @@ skale resources-allocation generate
...
 -   `--yes` - generate without additional confirmation
+-   `-f/--force` - rewrite allocation file if it exists
+
+### Validate commands
+
+> Prefix: `skale validate`
+
+#### Validate abi
+
+Check if abi failes present and have valid json data
```
Comment (cstrangedk) [inline README.md]: "Check whether ABI and JSON data is valid"

**Review by cstrangedk — CHANGES_REQUESTED** [review]

File: core/host.py [hunk 4]
```diff
@@ -112,3 +118,21 @@ def safe_mk_dirs(path, print_res=False):
...
+def validate_abi_files(json_result=False):
+    results = [
+        validate_abi(abi_filepath)
+        for abi_filepath in [
+            MANAGER_CONTRACTS_FILEPATH,
+            IMA_CONTRACTS_FILEPATH
+        ]
+    ]
+    if any(r['status'] == 'error' for r in results):
+        print('Some files are not exist or incorrect')
```
Comment (cstrangedk) [inline core/host.py]: 'Some files do not exist or are incorrect'

File: tests/cli/validate_test.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+def test_validate_abi(contract_valid_abi_files):
+    result = run_command(abi)
+    assert result.output == 'All abi files are correct json files!\n'
+    assert result.exit_code == 0
+
+
+def test_validate_abi_invalid_file(contract_abi_file_invalid):
+    result = run_command(abi)
+    assert result.output == 'Some files are not exist or incorrect\n                Filepath                   Status                 Msg              \n-----------------------------------------------------------------------------------\ntests/.skale/contracts_info/manager.json   error    Failed to load abi file as json\ntests/.skale/contracts_info/ima.json       ok                                      \n'  # noqa
```
Comment (cstrangedk) [inline tests/cli/validate_test.py]: Some files do not exist or are incorrect

File: tests/cli/validate_test.py [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    assert result.exit_code == 0
+
+
+def test_validate_abi_empty_file(contract_abi_file_empty):
+    result = run_command(abi)
+    assert result.output == 'Some files are not exist or incorrect\n                Filepath                   Status       Msg     \n----------------------------------------------------------------\ntests/.skale/contracts_info/manager.json   error    No such file\ntests/.skale/contracts_info/ima.json       ok                   \n'  # noqa
```
Comment (cstrangedk) [inline tests/cli/validate_test.py]: Some files do not exist or are incorrect

**Review by cstrangedk — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (badrogger) [reply README.md]: I think it's better to correct to "Check whether ABI files contain valid JSON data"

**Review by cstrangedk — APPROVED** [review]

**Pull request merged**